### DISCUSSION TITLE
Coquille dans le CSS

### DISCUSSION THREAD

**Discussion by StagiaireCafe** [Kim-Bnx/Blank-Theme#2] [General]
Bonjour ! :)
J'ai repéré une petite coquille dans le CSS : 
.award a pointer:cursor; au lieu de cursor:pointer;

**Comment by Kim-Bnx** [comment]
> Bonjour ! :) J'ai repéré une petite coquille dans le CSS : .award a pointer:cursor; au lieu de cursor:pointer;

Ah yes, et ça m'arrive souvent cette coquille en plus haha ! Merci du signalement, c'est corrigé ✌
